servers/rendering/renderer_rd/forward_clustered/render_fc.h

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  renderer_scene_render_forward_clustered.h                            */
+/*  render_fc.h                                                          */
```

Comment:
Why not name it "render_forward_clustered"? That is already way shorter than "Render_scene_render_forward_clustered"

Reply:
had to give it a name, I'm happy with it being a little more fully written out

Reply:
Though that would change `SceneShaderFC` into `SceneShaderForwardClustered` as well, which was why I abbreviated it to FC in the first  place

Reply:
Ok it's now `RenderForwardClustered` as the main class name but I've left `SceneShaderFC` seeing its only used for `RenderForwardClustered` so I think the name is clear.

Reply:
Actually I might rename a few more things for consistancy

Reply:
Ok, so we now have: `RenderForwardClustered` as the main class, and then `SceneShaderFC`, `RenderBufferDataFC` and `GeometryInstanceFC` as support classes/structs